src/components/InfoWindow/Contents/Tab.tsx

```diff
@@ -1,29 +1,31 @@
 import styled from '@emotion/styled';
 
-export const Tab = styled.div`
-	width: 100%;
-	height: 36px;
-	background: transparent;
-	display: flex;
-	gap: 3px;
+export const TabBar = styled.div`
+  width: 100%;
+  height: 36px;
+  background: transparent;
+  display: flex;
+  gap: 3px;
+  filter: brightness(0.99);
+  backdrop-filter: blur(8px);
+`;
 
-	div {
-		flex: 1;
-		display: flex;
-		align-items: center;
-		justify-content: center;
-		font-family: 'Pretendard';
-		font-style: normal;
-		font-weight: 600;
-		font-size: 14px;
-		background: #ffffff;
-		color: #999999;
-		border-radius: 12px 12px 0px 0px;
-		filter: brightness(0.99);
+export const Tab = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Pretendard';
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  background: #ffffff;
+  color: #999999;
```

Comment:
앞에 컴포넌트에서 props로 isSelected 전달해줬다면

```suggestion
  color: ${(props) => props.isSelected ? "black": "gray" };
  filter : ${(props) => props.isSelected ? "none" :  "(brightness(0.99)" };
```
혹은
```suggestion
  ${(props) => props. isSelected ? `${selectedStyle}`: `${defaultStyle}` }
```
이런식으로 쓸수 있을것같아요~